build.sbt

```diff
@@ -488,3 +487,22 @@ lazy val jmh = (project in file("jmh"))
     commonSettings,
   )
   .dependsOn(core, jena)
+
+
+lazy val grpc = (project in file("grpc"))
+  .settings(
+    name := "jelly-grpc",
+    description := "Implementation of a gRPC client and server for the Jelly gRPC streaming protocol.",
+    libraryDependencies ++= Seq(
+      "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5",
+      "org.apache.pekko" %% "pekko-actor-typed" % pekkoV,
+      "org.apache.pekko" %% "pekko-discovery" % pekkoV,
+      "org.apache.pekko" %% "pekko-stream-typed" % pekkoV,
+      "org.apache.pekko" %% "pekko-actor-testkit-typed" % pekkoV % Test,
+      "org.apache.pekko" %% "pekko-grpc-runtime" % pekkoGrpcV,
+    ),
+    commonSettings,
+  )
+  .dependsOn(stream % "test->compile")
+  .dependsOn(core % "compile->compile;test->test;protobuf->protobuf")
```

Comment:
When I run `sbt clean` I get

` java.lang.IllegalArgumentException: Cannot add dependency 'eu.neverblink.jelly#jelly-core;3.1.2+6-5fd98cf6-SNAPSHOT' to configuration 'protobuf' of module eu.neverblink.jelly#jelly-grpc_3;3.1.2+6-5fd98cf6-SNAPSHOT because this configuration doesn't exist!`